Composer/packages/lib/indexers/src/utils/luUtil.ts

```diff
@@ -104,13 +105,24 @@ export function convertLuParseResultToLuFile(id: string, resource, luFeatures: I
     convertLuDiagnostic(e, id)
   ) as Diagnostic[];
 
+  const imports = Sections.filter(({ SectionType }) => SectionType === SectionTypes.ImportSection).map(
+    ({ Path, Description }) => {
```

Comment:
Sections is just defined as `any[]`, so I can't tell if these fields actually exist. Would you mind tracking down the definition for what a "section" actually is and changing it accordingly in `packages/types/src/indexers.ts`? That's where the LuFile and LuParseResource types are defined.